jsapp/js/assetUtils.ts

```diff
@@ -528,8 +529,13 @@ export function getFlatQuestionsList(
         isRequired: Boolean(row.required),
         label: getQuestionOrChoiceDisplayName(row, translationIndex),
         path: flatPaths[rowName],
-        parents: openedGroups.slice(0),
-        hasRepatParent: openedRepeatGroupsCount >= 1,
+        parents: openedGroups
+          .slice(0)
+          .map((group) =>
+            getQuestionOrChoiceDisplayName(group, translationIndex)
+          ),
+        parentRows: openedGroups.slice(0),
```

Comment:
parents is for human reading only. Adding parentRows in addition feels a little arbitrary. I don't love it.